rust/src/storage/inmemory/notus_advisory.rs

```diff
@@ -0,0 +1,16 @@
+// SPDX-FileCopyrightText: 2025 Greenbone AG
+//
+// SPDX-License-Identifier: GPL-2.0-or-later WITH x11vnc-openssl-exception
+
+use crate::storage::{
+    dispatch::Dispatcher, error::StorageError, items::notus_advisory::NotusAdvisory,
+};
+
+use super::InMemoryStorage;
+
+impl Dispatcher<()> for InMemoryStorage {
+    type Item = NotusAdvisory;
+    fn dispatch(&self, _: (), item: Self::Item) -> Result<(), StorageError> {
```

Comment:
This is more of a general remark and not just relevant to NotusAdvisory. With this type system instead of runtime dispatching approach I cannot lookup where the functions are actually used within the actual code base by reference. It would be helpful to me if you could provide a grep string or something similar so that I can find out where the NotusAdvisory dispatch is actually used.

Reply:
It is not a general remark, rust-analyzer is actually able to find it this whole implementation is not used and can be deleted.